docs/src/doc/user-guide/methods.md

```diff
@@ -13,6 +13,12 @@ class RotatingCube: Spatial() {
 }
 ```
 
+!!! danger ""
```

Comment:
important is probably better here - because we give an important information not a warning for certain pitfalls.